createlinks

```diff
@@ -82,7 +83,8 @@ $event = "nethserver-firewall-base-save";
 event_templates($event, @templates);
 event_actions($event,
 	      'providers-cleanup' => '02',
-	      'nethserver-shorewall-restart' => '89'
+	      'nethserver-shorewall-restart' => '89',
+          'nethserver-firewall-base-rmmod-SipH323-modules' => '90'
```

Comment:
Change the name of the action: do not use upper case letters.

Reply:
done